src/NBS.Appointments.Service/Startup.cs

```diff
@@ -26,12 +27,18 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            var sessionManagerConfig = Configuration.GetSection("SessionManager").Get<SessionManagerOptions>();
             services
                 .Configure<QflowOptions>(Configuration.GetSection("Qflow"))
                 .Configure<DateTimeProviderOptions>(Configuration.GetSection("DateTimeProvider"))
                 .Configure<ApiKeyAuthenticationOptions>(options => options.ApiKey = Configuration.GetValue<string>("ApiKey"));
 
             services
+                .AddApplicationInsightsTelemetry()
+                .ConfigureTelemetryModule<EventCounterCollectionModule>((module, o) =>
```

Comment:
Can remove `.ConfigureTelemetryModule` (Line 38 to 41). Not required as app insights surfaces the cpu-usage metric.